SPECS-SIGNED/mstflint-driver-signed/mstflint-signed.spec

```diff
@@ -0,0 +1,35 @@
+%define mstflint_unsigned_name mstflint-kernel
+
+Summary:        Mellanox firmware burning tool
+Name:           %{mstflint_unsigned_name}-signed
+Version:        4.21.0
+Release:        4%{?dist}
+License:        Dual BSD/GPL
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Kernel
+URL:            https://github.com/Mellanox/%{name}
+Source0:        https://github.com/Mellanox/%{name}/releases/download/v%{version}-1/%{name}-%{version}-1.tar.gz
```

Comment:
In this case the source is only going to be the signed kernel module built from `mstflint.spec`.